apps/web/src/pages/ApplicantDashboardPage/components/ReviewApplicationPreviewList.tsx

```diff
@@ -0,0 +1,576 @@
+                : nullMessage}
+            </FieldDisplay>
+            <FieldDisplay
+              label={intl.formatMessage(commonMessages.salaryRange)}
+            >
+              {pool?.classification
+                ? getSalaryRange(locale, pool.classification)
+                : nullMessage}
+            </FieldDisplay>
+            <FieldDisplay
+              label={intl.formatMessage(talentRequestMessages.workStream)}
+            >
+              {pool.workStream?.name?.localized}
+            </FieldDisplay>
+            <FieldDisplay
+              label={intl.formatMessage(processMessages.languageRequirement)}
+            >
+              {pool.language?.label.localized}
+            </FieldDisplay>
+            <FieldDisplay
+              label={intl.formatMessage(commonMessages.department)}
+              data-h2-grid-column="p-tablet(span 2)"
+            >
+              {pool.department?.name?.localized}
+            </FieldDisplay>
+            <FieldDisplay
+              label={
+                showDeadlineToApply ? (
+                  <span data-h2-color="base(black)">
+                    {intl.formatMessage(commonMessages.deadlineToApply)}
+                  </span>
+                ) : (
+                  intl.formatMessage(commonMessages.received)
+                )
+              }
+              data-h2-grid-column="p-tablet(span 2)"
+              hasError={showDeadlineToApply}
+            >
+              {showDeadlineToApply ? (
+                <>
+                  {pool?.closingDate
+                    ? formatDate({
+                        date: parseDateTimeUtc(pool?.closingDate),
+                        formatString: "PPP",
+                        intl,
+                        timeZone: "Canada/Pacific",
+                      })
+                    : nullMessage}
+                </>
+              ) : (
+                <>
+                  {application.submittedAt
+                    ? formatDate({
+                        date: parseDateTimeUtc(application.submittedAt),
+                        formatString: "PPP",
+                        intl,
+                        timeZone: "Canada/Pacific",
+                      })
+                    : nullMessage}
+                </>
+              )}
+            </FieldDisplay>
+
+            <Separator
+              decorative
+              data-h2-grid-column="p-tablet(span 2)"
+              data-h2-margin="base(0)"
+            />
+
+            <FieldDisplay
+              label={intl.formatMessage(commonMessages.employmentLength)}
+              data-h2-grid-column="p-tablet(span 2)"
+            >
+              {pool.workStream?.name?.localized}
+            </FieldDisplay>
+            <FieldDisplay
+              label={intl.formatMessage(talentRequestMessages.workLocation)}
+              data-h2-grid-column="p-tablet(span 2)"
+            >
+              {pool.location?.localized}
```

Comment:
It looks like pools considered remote won't display anything for location.
![image](https://github.com/user-attachments/assets/be38814d-d8a2-4c88-be8a-29ddd8e43c86)

Reply:
Fixed in [6faad8c](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/6faad8ce0d342369bcbff089b97c7905dce2f780)